txmatching/data_transfer_objects/external_patient_upload/swagger.py

```diff
@@ -80,12 +80,15 @@
     }
 )
 
+HlaToAntibodyMatchJson = public_api.model('HlaToAntibodyMatch', {
```

Comment:
```suggestion
HLAToAntibodyMatchJson = public_api.model('HlaToAntibodyMatch', {
```
+ Případně změnit podle nového názvu HLAToAntibodyMatch